server/main.lua

```diff
@@ -109,11 +109,16 @@ RegisterNetEvent('hospital:server:SetLaststandStatus', function(bool)
 end)
 
 RegisterNetEvent('hospital:server:SetArmor', function(amount)
-	local src = source
-	local Player = QBCore.Functions.GetPlayer(src)
-	if Player then
-		Player.Functions.SetMetaData("armor", amount)
-	end
+    local src = source
+    local Player = QBCore.Functions.GetPlayer(src)
+    if Player then
+        local newArmor = amount
```

Comment:
This variable newAmount os unnecessary, you can just use amount directly as it's already local

Reply:
so newArmor -> amount?

Reply:
Yup

Reply:
Did I do it right? lmao

Reply:
Yes you did